src/otx/core/model/module/base.py

```diff
@@ -53,6 +54,7 @@ def __init__(
         torch_compile: bool,
         optimizer: list[OptimizerCallable] | OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
         scheduler: list[LRSchedulerCallable] | LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
+        metric: Metric,
```

Comment:
Thanks for the suggestion, Making the `metric=None` doesn't looks good to me since metric should be delivered (So, can't be None).  As you said, all tasks initialize the `metric` again, I don't want to cause any confusion to the readers that the metric could be None.